.chloggen/mx-psi_update-tov0.11.0.yaml

```diff
@@ -7,15 +7,22 @@ change_type: enhancement
 component: datadogexporter
 
 # A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
-note: Add `datadog.otlp_translator.metrics.missing_source` counter, which counts the number of metrics that are missing a source (e.g. hostname). 
```

Comment:
Is it expected that the old entry be dropped?

Reply:
Yes, it's intentional: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c88b4312804f5a42eed3615a9e70e6d65ce4f9d3/exporter/datadogexporter/factory.go#L100-L102 we need to discuss internally some details about the metric, so I am dropping that changelog note for now